### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by secana**
- Upgrade the project to .NET 5
- Add docs for the setup and configuration of MSE

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -39,20 +120,152 @@ kubectl apply/delete -f k8s-deployment.yaml
...
+import hashlib
+import pymongo # sudo pip install pymongo
+import datetime
+import os
+import sys, getopt
+
+def put_string_into_db(sha256, platform, file_size, sample_set, mongo_collection):
+    current_iso_datetime = datetime.datetime.utcnow()
+    entry = {
+                "_id": f"{sha256}:test",                  # Unique ID
```
Comment (secana) [inline Readme.md:210]: I'm not sure where the ":test" comes from. Does somebody know, why it's needed and what value should be set here?

Comment (unglaublicherdude) [reply Readme.md:210]: This stands for the sample-set. If the test-data is just filled with this one sample-set you could also just remove it. If you leave it there, maybe rename it to testSampleSet

**Review by unglaublicherdude — APPROVED** [review]

File: Readme.md (hunk 1 above)
Comment (secana) [reply Readme.md:210]: So shouldn't it be the same value as the "SampleSet" property below?

File: Readme.md [hunk 2]
```diff
@@ -1,7 +1,85 @@
...
+actual payload that contains expiration date of the token, SHA256, file size and the platform. The third part is a signature that guarantees that the JWT is valid.
+If after checking that the sample is not already part of your collection and you have interest in the reported platform,
+you can download it with: 
+
+```bash
+# Download a specific sample
+curl -X GET -k "http://localhost:8080/v1/download?token=$PUT_TOKEN_HERE"
+```
+
+Not authentication is needed for the download, as the [JWT](https://jwt.io/) is signed and as such authenticates the request.
```
Comment (bastetfurry) [inline Readme.md:55]: s/Not authentication/No authentication/;

Comment (secana) [reply Readme.md]: fixed

**Review by bastetfurry — APPROVED** [review]

**Pull request merged**